### PR TITLE
Explicit check if the msi stock id is set for the given storeview

### PR DESCRIPTION
**Pull request by philippsander**

### CODE REVIEW

File: src/api/stock.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default ({ config, db }) => {
 
   const _getStockId = (storeCode) => {
     let storeView = config.storeViews[storeCode]
-    return storeView ? storeView.msi.stockId : config.msi.defaultStockId
+    return storeView  && storeView.msi && storeView.msi.stockId ? storeView.msi.stockId : config.msi.defaultStockId
```
Comment (ResuBaka) [inline src/api/stock.ts:15]: As we have typescript version 3.7 available we can use [Optional Chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) which in my opinion is much cleaner when the "&&" chain. 

```suggestion
    return storeView?.msi?.stockId ? storeView.msi.stockId : config.msi.defaultStockId
```

**Comment by gibkigonzo** [comment]
@philippsander Can you check suggestion and update changelog? :)

**Comment by philippsander** [comment]
@gibkigonzo Sorry for being late! i will update the changelog

**Review by philippsander — COMMENTED** [review]
Suggestion applied. Thank you!

**Comment by philippsander** [comment]
@ResuBaka Anything else I should do?

**Review by gibkigonzo — APPROVED** [review]

**Review by ResuBaka — APPROVED** [review]

**Comment by ResuBaka** [comment]
All fine. Thanks for the fix.

**Pull request merged**